tree/treeplayer/test/dataframe/dataframe_cache.cxx

```diff
@@ -163,3 +191,62 @@ TEST(Cache, NonCopiable)
    EXPECT_EQ(0, ret)
       << "The static assert was not triggered even if caching of columns of a non copiable type was requested";
 }
+
+auto treeName = "t";
+auto fileName = "fileName.root";
```

Comment:
I guess this was a placeholder name

Reply:
it's to avoid to copy&paste strings.

Reply:
Yes but `fileName.root` is a bit anonymous so I thought you meant to change it later :sweat_smile:

Reply:
oh, I did not notice that. You are right: that name is bogus and would cause trouble. Will change.